### PR TITLE
fix: dispose quill keyboard handler before block selection

### PR DESCRIPTION
**Pull request by thorseraq**

### CODE REVIEW

File: packages/blocks/src/__internal__/rich-text/rich-text-operations.ts [hunk 1]
```diff
@@ -224,18 +224,15 @@ export function handleLineStartBackspace(page: Page, model: ExtendedModel) {
...
     const index = parent.children.indexOf(model);
+    const blockProps = {
+      flavour: 'affine:paragraph',
+      type: 'text',
+      text: model.text?.clone(),
+      children: model.children,
+    };
     page.captureSync();
     page.deleteBlock(model);
-    const id = page.addBlockByFlavour(
```
Comment (himself65) [inline packages/blocks/src/__internal__/rich-text/rich-text-operations.ts:229]: why dont use this?

Comment (thorseraq) [reply packages/blocks/src/__internal__/rich-text/rich-text-operations.ts:229]: It's will cause unit test fail

Comment (himself65) [reply packages/blocks/src/__internal__/rich-text/rich-text-operations.ts:229]: Seems like a bug

Comment (himself65) [reply packages/blocks/src/__internal__/rich-text/rich-text-operations.ts:229]: Could you please create a issue?

**Review by doodlewind — APPROVED** [review]

**Pull request merged**